packages/fast-components-react-msft/app/app.tsx

```diff
@@ -56,6 +67,12 @@ export default class App extends React.Component<{}, {}> {
     }
 
     private handleUpdateDirection = (direction: Direction): void => {
```

Comment:
how is this `handleUpdateDirection` used? It seems like "direction" should really just be a prop - this function doesn't get called anywhere either

Reply:
Synced offline - not seen b/c the code was folded.